### PR TITLE
Add test cases for emqx_dashboard_cli

### PR DESCRIPTION
**Pull request by linjunjj**
Add test cases for emqx_dashboard_cli

### CODE REVIEW

File: test/emqx_dashboard_cli_SUITE.erl [hunk 1]
```diff
@@ -4,11 +4,70 @@
...
-    [].
+init_per_suite(Config) ->
+    [start_apps(App, {SchemaFile, ConfigFile}) ||
+        {App, SchemaFile, ConfigFile}
+            <- [{emqx, local_path("deps/emqx/priv/emqx.schema"),
+                       local_path("deps/emqx/etc/emqx.conf")},
+                {emqx_management, local_path("deps/emqx_management/priv/emqx_management.schema"),
+                                  local_path("deps/emqx_management/etc/emqx_management.conf")},
+                {emqx_dashboard, local_path("priv/emqx_dashboard.schema"),
+                                 local_path("etc/emqx_dashboard.conf")}]],
```
Comment (Gilbert-Wong) [inline test/emqx_dashboard_cli_SUITE.erl]: Duplicate with the procedure in another SUITE. I suggest you to abstract the common logic.

File: test/emqx_dashboard_cli_SUITE.erl [hunk 2]
```diff
@@ -4,11 +4,70 @@
...
+        true -> ok;
+        false -> ct:fail("password error")
+    end,
+    emqx_dashboard_cli:admins(["passwd", "username", "newpassword"]),
+    [{mqtt_admin, <<"username">>, <<Salt1:4/binary, Hash1/binary>>, _}] = 
+        emqx_dashboard_admin:lookup_user(<<"username">>),
+    case Hash1 =:= erlang:md5(<<Salt1/binary, <<"newpassword">>/binary>>) of
+        true -> ok;
+        false -> ct:fail("password error")
+    end,
```
Comment (terry-xiaoyu) [inline test/emqx_dashboard_cli_SUITE.erl]: ?assertEqual(Hash1, erlang:md5(<<Salt1/binary, <<"newpassword">>/binary>>))

File: test/emqx_dashboard_cli_SUITE.erl [hunk 3]
```diff
@@ -4,11 +4,70 @@
...
+    emqx_dashboard_cli:admins(["del", "username"]),
+    [] = emqx_dashboard_admin:lookup_user(<<"username">>),
+    emqx_dashboard_cli:admins(["add", "admin1", "pass1"]),
+    emqx_dashboard_cli:admins(["add", "admin2", "passw2"]),
+    AdminList = emqx_dashboard_admin:all_users(),
+    2 = length(AdminList).
```
Comment (terry-xiaoyu) [inline test/emqx_dashboard_cli_SUITE.erl]: ?assertEqual()

File: test/emqx_dashboard_cli_SUITE.erl [hunk 4]
```diff
@@ -4,11 +4,70 @@
...
+        emqx_dashboard_admin:lookup_user(<<"username">>),
+    case Hash1 =:= erlang:md5(<<Salt1/binary, <<"newpassword">>/binary>>) of
+        true -> ok;
+        false -> ct:fail("password error")
+    end,
+    emqx_dashboard_cli:admins(["del", "username"]),
+    [] = emqx_dashboard_admin:lookup_user(<<"username">>),
+    emqx_dashboard_cli:admins(["add", "admin1", "pass1"]),
+    emqx_dashboard_cli:admins(["add", "admin2", "passw2"]),
+    AdminList = emqx_dashboard_admin:all_users(),
```
Comment (Gilbert-Wong) [inline test/emqx_dashboard_cli_SUITE.erl]: Please import eunit  and assert two expressions.

**Pull request merged**